src/Entity/Document.php

```diff
@@ -63,6 +63,11 @@ class Document implements LoggableEntityInterface
      */
     private $hearingPost;
 
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $path;
```

Comment:
A comment on what `path` actually contains could come in handy at some point.

Reply:
Added comment.